actors/builtin/multisig/multisig_state.go

```diff
@@ -58,6 +58,63 @@ func (st *State) AmountLocked(elapsedEpoch abi.ChainEpoch) abi.TokenAmount {
 	return locked
 }
 
+// Iterates all pending transactions and removes an address from each list of approvals, if present.
+// If an approval list becomes empty, the pending transaction is deleted.
+func (st *State) PurgeApprovals(store adt.Store, addr address.Address) error {
```

Comment:
We should wrap all the myriad of different errors down here to add information about what went wrong.